Ferlab-Ste-Justine/clin-frontend
# feat: #235 avoid creating grand parents

# [FEATURE] [Do not allow to create grandparents]

closes #[235]

## Description
for a given family, the UI must block the possibly of adding a grandparent

## Validation


## Screenshot 
![screenshotCannotAddParentGalileo](https://user-images.githubusercontent.com/54366437/134405674-b6a27ab4-8c7a-4557-bc1e-1748b1253a3d.png)

## QA
Create a new patient, add a father and or a mother. After that, UI should block you to create another member.

Note: some old data can give weird behaviours. pls tests on new patients only.
